src/domain/session/rightpanel/RoomInfoViewModel.js

```diff
@@ -0,0 +1,50 @@
+import { ViewModel } from "../../ViewModel.js";
+import { avatarInitials, getIdentifierColorNumber, getAvatarHttpUrl } from "../../avatar.js";
+
+export class RoomInfoViewModel extends ViewModel {
```

Comment:
can we call this `RoomDetails{View|ViewModel}` everywhere? It's consistent with the details segment in the nav/url. Also the CSS classes, see comments in `element/theme.css`